src/commands/moderation/unwarn.ts

```diff
@@ -40,12 +40,11 @@ export default class Unwarn {
     const name = selectedMember.nickname ?? user.username;
     const dmChannel = (await user.createDM().catch(() => null)) as DMChannel | null;;
     const id = interaction.options.getNumber("id", true);
-    const warns = await modTable?.get(`${interaction.guild.id}.${user.id}.warns`).then(
-      warns => warns as any[] ?? []
-    ).catch(() => []);
+    const warns = await modTable
+      ?.get(`${interaction.guild.id}.${user.id}.warns`)
```

Comment:
```suggestion
    const warns = await modTable?
      .get(`${interaction.guild.id}.${user.id}.warns`)
```
That character belongs there I think ;)